setup.py

```diff
@@ -6,7 +6,7 @@ def read(*rnames):
     return open(os.path.join(*rnames)).read()
 
 long_description = (
-    read('src', 'z3c', 'relationfield', 'README.txt')
+    read('src', 'z3c', 'relationfield', 'README.rst')
     + '\n' +
     read('CHANGES.txt')
```

Comment:
Absolutely. Forgot about grep'ing for CHANGES.rst.
Thanks for the pointer - updated.